_projects/2022/100376720/100376720.Rmd

```diff
@@ -0,0 +1,401 @@
+---
+title: "Education Attainment Graph"
+description: |
+ This project consists in the replicaion and alternative representation of a graph from a New York Times article called "Where Education Drives Mobility".
+author: Edgar Chicurel
+date: "`r Sys.Date()`"
+output:
+  distill::distill_article:
+    self_contained: false
+    Add toc: true
+preview: original.png
+---
+
+## Introduction
+
+The graph for this project was inspired by a New York Times article about education mobility and gender differences in education. The first part of the article talks about how young students' highest level of education compares to their parents' highest level of education. Students who achieved a higher level than their parents are considered upwardly mobile, those who achieved the same level are considered to have experienced status quo mobility, and those who achieved a lower level are considered downward mobile. The second part of the article examines gender differences in education at a specific level. You can find the full article [here](https://www.nytimes.com/2019/06/07/education/learning/bulletin-board-graduation-guidance.html).
+
+
+## Graph to replicate (two graphs in one)
+
+The graph that needs to be replicated is actually two graphs in one. The first graph (on the left) shows the differences in education mobility among OECD countries for students aged 25-34. The graph uses a dark purple color for students who achieved a higher education level than their parents, a light grey color for students who achieved the same level, and a yellow color for students who achieved a lower level. The second graph (on the right) shows the difference for the same countries, except for Russia and Japan (for which data is not available), but by country and gender for students who achieved a highest education level higher than high school. The first graph is a stacked percentage bar graph, and the second graph consists of dotted lines and points. Both graphs have annotations to label the categories. At first glance, the graphs do not appear to be complicated. Here is a link to the original graph:
+
+![Graph to replicate](original.png){.external width="100%"}
+
+
+## Load libraries and read the database
+
+The first step was to load the libraries, in this case I only used two: tidyverse and patchworks. The second step was to read the database which was on a csv file. 
+
+
+```{r, setup, results="hide"}
+
+library(tidyverse)
+library(patchwork)
+
+df <- read_csv("C:/Users/usuario/Desktop/Master/Data Visualization/Data_on_intergenerational_mobility.csv")
```

Comment:
As with the original image, please do the same with the dataset. Move the dataset next to the Rmd, and then read it just by name `read_csv("Data_on_intergenerational_mobility.csv")`.